src/etc/module.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Superpayments_SuperPaymentHyva" setup_version="2.6.0">
+    <module name="Superpayments_SuperPaymentHyva" setup_version="2.6.1">
```

Comment:
The `setup_version` in module.xml and the version in composer.json serve different purposes. Syncing them is unnecessary and can cause confusion. Changing the `setup_version` in module.xml signals Magento to run database schema updates when the `bin/magento setup:upgrade` command is executed. There are no actual database schema or data changes, this leads to unnecessary overhead during deployment.